### PR TITLE
#162052401 Implement Logging on Kubernetes and the Sandbox

### PR DESCRIPTION
**Pull request by WinstonKamau**
##### What does this PR do?
This Pull Request implements logging on the frontend application.

##### Description of Task to be completed?
The task involves installing a package from npm store called [morgan](https://www.npmjs.com/package/morgan). The package is a HTTP request logger middleware for node.js. The logger is directed to log to a file on the sandbox to enable use of volumes. It maintains its default logging on the GCP infrastructure.

##### Any background context you want to provide?
None

##### What are the relevant Pivotal Tracker stories?
https://www.pivotaltracker.com/story/show/162052401

##### Screenshots?
A view of logs from the sandbox 
image
A view of logs on GCP
image

### CODE REVIEW

File: docker/dev/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#due to the rapid changes the serve team is making please 
+#use this version of serve
+
+RUN yarn global add express@4.16.2 morgan@1.9.1
+
+WORKDIR /application
+
+COPY public /application
+
+COPY sandserver /application
```
Comment (mrmuli) [inline docker/dev/Dockerfile]: those copy layers can be nested

File: docker/dev/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+ENV NODE_PATH /usr/local/share/.config/yarn/global/node_modules/
```
Comment (mrmuli) [inline docker/dev/Dockerfile]: this could move further up the initial layers

File: docker/dev/sandbox.md [hunk 3]
```diff
@@ -94,4 +94,4 @@ On the root of the backend repository you will be able to find application logs
 
 #### Frontend Logs 
 
-Coming soon!!
\ No newline at end of file
+On the root of the frontend repository you will be able to find application logs under the logs folder. This folder is created after starting the sandbox. This folder contains one file application.log. Application errors and access logs are logged to the application.log file.
```
Comment (mrmuli) [inline docker/dev/sandbox.md]: can we discuss the implementation of the frontend logs?

File: docker/dev/Dockerfile (hunk 2 above)
Comment (WinstonKamau) [reply docker/dev/Dockerfile]: I have fixed this.

**Comment by WinstonKamau** [comment]
@mrmuli I have fixed everything on this P.R as per our sync.

**Pull request merged**